src/test/java/core/basesyntax/db/ReportServiceImplTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.db;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.service.ReportService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceImplTest {
+    private ReportService reportService;
+
+    @BeforeEach
+    void setUp() {
+        reportService = new ReportServiceImpl();
+    }
+
+    @Test
+    void returnsCorrectData_Ok() {
```

Comment:
```suggestion
    void generate_correctData_Ok() {
```
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/20326858/acd63817-8b3d-4a4d-a66b-22be9106baa6)